src/components/ProgressBar/index.scss

```
@@ -0,0 +1,36 @@
+@use './../../styles/base.scss' as *;
+
+.progress-bar {
+  width: 100%;
+
+  margin: 0;
+  padding-top: 20px;
+  padding-right: 40px;
+  padding-left: 40px;
```

Usamos REM para unidades, tirando bordas e borde-radius.

E esse padding poderia ter escrito em uma linha só 
```suggestion
padding: 2rem 4rem 2rem;
```

E esse padding é necessário?